packages/store/src/store.js

```diff
@@ -12,26 +12,26 @@ import { insist } from '@agoric/insist';
  */
 function makeStore(keyName = 'key') {
   const store = new Map();
-  const insistKeyDoesNotExist = key =>
-    insist(!store.has(key))([`${keyName} already registered`]);
-  const insistKeyExists = key =>
-    insist(store.has(key))([`${keyName} not found: `, ''], key);
+  const assertKeyDoesNotExist = key =>
+    assert(!store.has(key))([`${keyName} already registered`]);
```

Comment:
This seems like it cannot be right because assert does not return a function. What's going on here?